### ISSUE TITLE
Submit current directory

### ISSUE THREAD

**Issue by morhekil** [GoChallenge/cli#2]
While playing with cli tool, it was initially quite confusing to me that it was always running tests in the default package directory (where it downloaded the challenge code), and I was in (and trying to submit) another directory.

I do wonder if other people might want to submit code from non-standard directories, too, and whether we shall just submit the current directory, instead of a fixed pre-defined location? I'd love to hear others' opinions on this.

**Comment by idawes** [comment]
I was originally slightly confused by it too, but I've come around to the simplicity of it. It kind of fits with Go, there's no real reason why the tool should support using a different directory. I think that as long as we write up a very good description of exactly what the process is, this will make participating in the challenge really simple.

**Comment by nindalf** [comment]
@morhekil The first version actually supported choosing a directory or a set of files and the archive function would react accordingly. But submitting code directly from the challenge directory simplifies the code and the usage of the tool.

It does make it more difficult to test, so we could do something about that.

**Comment by morhekil** [comment]
I agree that asking a user for / choosing a directory would complicate things to much. But why not just use the current one? I feel it is natural, when running "gochallenge submit" while being in a specific directory, that this is exactly what is being submitted. Think of other basic directory tools, like even `ls` or `tree` - when an explicit directory is not given, they operate on the current directory, not user's home directory or anything else.

**Comment by nindalf** [comment]
`ls` is a general tool that can be used on any directory. `submit` will only work on one directory - the current challenge. We should reconsider this when we decide to allow older challenges to be submitted. Currently for a person who has the tool installed, its a simple flow.

`gochallenge fetch`
// code for some time
`gochallenge submit`
